static/css/devhub/new-landing/base.less

```diff
@@ -65,6 +65,23 @@ img {
   max-width: 100%;
 }
 
+.link(@color, @decoration: none) {
+  color: @color;
+  text-decoration: @decoration;
+
+  &:visited {
+    color: fadeOut(@color, 10%);
+  }
+
+  &:hover {
```

Comment:
Nice reference, thanks! I'll add underlines and refactor the mixin a bit more to be separated from our button.